docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -109,16 +109,65 @@ Follow the instructions below based on the query type:
-9. Enter SLO Details, including a **Name** and **Description**. This is used in the list and for searches.<br/><img src={useBaseUrl('img/observability/slo-create-details.png')} alt="Reliability Management SLO SLI" />
+   * **Target**. The value in percentage you want to target for the SLO (for example, you'd enter `99` for 99%).
+   * **Compliance Type**. **Rolling** provides a sequence of recent days for the Compliance Period, such as the last 7d or last 30d. **Calendar** calculates over a window of time for a Week.
+   * **Timezone**. Selecting a timezone is important to accurately assign events on the boundary of a compliance period, such as events received at 11:59 PM in a particular time zone.<br/><img src={useBaseUrl('img/observability/slo-create-slo.png')} alt="Reliability Management SLO SLI" width="400"/>
+9. Enter SLO Details:
+   * **Name**. Name of the SLO.
+   * **Description**. Short explanation of your SLO.
+   * **Tags**. Tags are key/value pairs that allow you to apply additional metadata to your SLOs beyond the name and description. [Learn more here](#slo-tags-and-filters). Tags can help you categorize, search, and filter your SLOs.<br/>Select the key and its possible values from the tags dropdown, or create your own key/value pair for the tag.<br/>
+  :::info
+  You can associate multiple tags with your SLO.
+  :::
+  <img src={useBaseUrl('img/observability/slo-details.png')} alt="SLO Details" width="800"/>
 10. Click **Save**. To create a monitor, click [Save and Create Monitor](#create-an-slo-monitor).
 
 
+### SLO Tags and Filters
+
+You can add key/value pair tags to your SLOs to allow you to better organize and filter them. You might find it useful to add tags for `team`, `service`, and `application`, for example.
+
+#### Limitations
+
+- Tag keys cannot start with the prefixes `sumo.` or `_`
+- Tag keys can only contain letters, numbers, `_`, `.`, `/`, `+`, `-`, `@`
+- Tag values can only contain letters, white space, numbers, `_`, `.`, `/`, `=`, `+`, `-`, `@`
+
+You can associate a maximum of 50 tags per SLO.
+
+#### Add a Tag
+
+To add a tag(s) to an existing SLO:
+
+1. In Sumo Logic, click **Manage Data** > **Monitoring** > **SLOs** tab.
+1. Click on any SLO line item in your list, then click **Edit**.
+1. Scroll down to section **(3) SLO Details**. Click on **Tags (Optional)** and type in a new tag or select an existing tag.
+1. Click **Save**.
+
+To add a tag while creating a new SLO:
+
+1. In Sumo Logic, click **Manage Data** > **Monitoring** > **SLOs** tab.
+1. Click **Add** > **New SLO**.
+1. After you've filled out sections **1** and **2**, scroll down to section **(3) SLO Details**. Click on **Tags (Optional)** and type in a new tag or select an existing tag.
+1. Click **Save**.
+
+#### Filter SLOs By Tags
```

Comment:
We should add a section for "Tags in SLO lookup table". Tags are available under SLO lookup table. Users will be able to see tags for SLOs under lookup table content. Screenshot (query [link](https://epddata.stag.sumologic.net/ui/#/search/create?id=cmcX2e0hjZXOl7EoQSBYslMHUi2dJd4XUAR0fjmW)):

<img width="1454" alt="image" src="https://github.com/SumoLogic/sumologic-documentation/assets/13195152/8cb5e730-54ae-477a-912f-576a396e9924">
 
To see all the SLOs with at least one tag ([link](https://epddata.stag.sumologic.net/ui/#/search/create?id=bKESghyFSBsxeeEkQHNhH4qTKE0j2N4ZD6j1OHhe)):

<img width="1437" alt="image" src="https://github.com/SumoLogic/sumologic-documentation/assets/13195152/cfcf66d3-1848-4a8a-b879-c7a0015be8a0">

Users can use tags in this lookup table for correlation with other data in sumo logic. e.g. they can use it for finding SLO output data for all SLOs that belong to service `ingestion` using the following query:

_view=sumologic_slo_output | lookup tags from sumo://content/slos on sloId=sloId | json field=tags "service" | where service="ingestion"